### ISSUE TITLE
Dimension whitelist / blacklist (MC1.12.2)

### ISSUE THREAD

**Issue by drivetheory** [MatrexsVigil/harvestcraft#201]
I've been unable to locate any documentation about  whitelisting or blacklist dimensions Pam's can / cannot generate in. The closest I've found to the information i am looking for is a post from 3 years ago for MC 1.7.10...
https://minecraft.curseforge.com/projects/pams-harvestcraft/files/2228552

In the future, some type of config option to make harvestcraft generation by dimension an opt-in setting instead of opt-out setting would be fantastic. However, until then, is there some type of setting that can be used to stop Pam's from generating in other dimensions? (ie. The Twilight Forest, The Betweenlands, etc.)

**Comment by fllo** [comment]
Which harvestcraft version are you using? 1.12.2t?
What type generates in other dimensions?
* gardens
* fruittrees
* beehives

**Comment by ghost** [comment]
Latest Update of 1.7.10 is
https://minecraft.curseforge.com/projects/pams-harvestcraft/files/2270206

To allow spawning in other biomes,
It is important to know which biomome to use.

In the configuration file [harvestcraft.cfg] lines 29 to 32 must be set to true.

**Comment by drivetheory** [comment]
@fllo 
"Pam's HarvestCraft 1.12.2q"
(which is the version Direwolf20 has in his latest release, "FTBPresentsDirewolf20112-1.10.0-1.12.2.zip")
https://www.feed-the-beast.com/projects/ftb-presents-direwolf20-1-12/files
so far i have witnessed fruit trees & gardens both spawning in other dimensions, beehives i have not seen "yet".

@PCobald
I am using MC v1.12.2, not 1.7.10, the only vague documentation i have seen for the feature i am looking for was a reference to an update to 1.7.10, which i listed in my first post.....

**Comment by fllo** [comment]
In 1.12.2t fruit trees are using whitelisted biomes only.
For each fruit you can white list the biomes that the fruittree is allowed to generate in.
Can be configured in harvestcraft_fruittree.cfg

So regarding fruittrees 1.12.2t should work.
But garden and beehive generation behavior hasn't changed yet.

**Comment by SnowShock35** [comment]
Having the dimension blacklisting for gardens and beehives would be great. It's quite frustrating not being able to prevent a certain block from spawning in a dimension.  😄

**Comment by drivetheory** [comment]
this feature is still needed...

**Comment by Petschko** [comment]
Yeah its annoying to have a flat creative world with just the Gardens... A Dim-Blacklist would be appreciated^^

**Comment by JomSpoons** [comment]
I would also appreciate this feature, especially if it could be added to 1.12.2 as well
